### DISCUSSION TITLE
cnf_setup error "cast from Int64 to String failed, at /usr/share/crystal/src/yaml/any.cr:168:5:168"

### DISCUSSION THREAD

**Discussion by mahantesh97** [lfn-cnti/testsuite#1801] [Q&A]
cnf-testsuite cnf_setup cnf-config=./cnf-testsuite.yml
Successfully created directories for cnf-testsuite
cnf-testsuite namespace already exists on the Kubernetes cluster
cnf setup online mode
/usr/share/crystal/src/yaml/any.cr:168:5 in '??'
/workspace/lib/totem/src/totem/config_types.cr:68:11 in 'cast_to_any_hash'
/workspace/lib/totem/src/totem/config_types/yaml.cr:7:7 in 'from_file'
/workspace/src/tasks/utils/cnf_manager.cr:518:14 in 'generate_and_set_release_name'
/workspace/src/tasks/utils/config.cr:47:16 in 'parse_config_yml'
/workspace/src/tasks/utils/config.cr:36:5 in 'sample_setup'
/usr/share/crystal/src/colorize.cr:132:3 in '->'
/workspace/lib/sam/src/sam/execution.cr:20:7 in '__crystal_main'
/usr/share/crystal/src/crystal/main.cr:115:5 in 'main'
src/env/__libc_start_main.c:95:2 in 'libc_start_main_stage2'
cast from Int64 to String failed, at /usr/share/crystal/src/yaml/any.cr:168:5:168

**Comment by agentpoyo** [comment]
Please see: https://github.com/cncf/cnf-testsuite/pull/1800

**Comment by mahantesh97** [comment]
Upgrading to latest HELM resolved the issue.

https://github.com/cncf/cnf-testsuite/blob/main/sample-cnfs/sample-coredns-cnf/README.md
#####ubuntu@zauk57asepp01dir001:~/cncf$ curl -fsSL -o get_helm.sh https://raw.githubusercontent.com/helm/helm/master/scripts/get-helm-3
ubuntu@abc~/cncf$ ls
cluster_tools.yml  cnf-testsuite.yml  cnfs  get_helm.sh  points.yml  results  tools
ubuntu@abc:~/cncf$ chmod 700 get_helm.sh
ubuntu@abc:~/cncf$ ./get_helm.sh
Helm v3.12.1 is available. Changing from version v3.10.1.
Downloading https://get.helm.sh/helm-v3.12.1-linux-amd64.tar.gz
Verifying checksum... Done.
Preparing to install helm into /usr/local/bin
helm installed into /usr/local/bin/helm


###ubuntu@abc:~/cncf$ cnf-testsuite cnf_setup cnf-config=./cnf-testsuite.yml
Successfully created directories for cnf-testsuite
cnf-testsuite namespace already exists on the Kubernetes cluster
cnf setup online mode
Successfully setup coredns
cnf setup online mode complete
